ape_aws/kms/_cli.py

```diff
@@ -54,7 +63,103 @@ def create_key(
     cli_ctx.logger.success(f"Key created successfully with ID: {key_id}")
 
 
-# TODO: Add `ape aws kms import`
+@kms.command(name="import")
+@ape_cli_context()
+@click.option(
+    "-p",
+    "--private-key",
+    "private_key_path",
+    type=click.Path(),
+    help="The private key you intend to import",
+    metavar="Path",
+)
+@click.option(
+    "-a",
+    "--admin",
+    "administrators",
+    multiple=True,
+    help="Apply key policy to a list of administrators if applicable, ex. -a ARN1, -a ARN2",
+    metavar="list[ARN]",
+)
+@click.option(
+    "-u",
+    "--user",
+    "users",
+    multiple=True,
+    help="Apply key policy to a list of users if applicable, ex. -u ARN1, -u ARN2",
+    metavar="list[ARN]",
+)
+@click.option(
+    "-d",
+    "--description",
+    "description",
+    help="The description of the key you intend to create.",
+    metavar="str",
+)
+@click.option(
+    "--use-mnemonic",
+    "import_from_mnemonic",
+    help="Import a key from a mnemonic phrase",
+    is_flag=True,
+)
+@click.option(
+    "--hd-path",
+    "hd_path",
+    help="The hierarchical deterministic path to derive the key from",
+    metavar="str",
```

Comment:
Don't need metavar here, but can set a default to help
```suggestion
    default=ETHEREUM_DEFAULT_PATH,
```